### ISSUE TITLE
Reprecudebility of the Cora results with GraphUNet

### ISSUE THREAD

**Issue by dvirginz** [pyg-team/pytorch_geometric#893]
Hi,
I'm trying to reproduce the results of splineNet on the cora dataset.
As the author of the paper, and as this is the official repo of the paper, can I please ask if the implementation provided [here](https://github.com/rusty1s/pytorch_geometric/blob/c5015cf3fe3e1335f3b18c712d308ee00de20dc0/examples/cora.py) should get 0.88?
I'm running the same script, but without these lines:
```
data.train_mask = torch.zeros(data.num_nodes, dtype=torch.bool)
data.train_mask[:data.num_nodes - 1000] = 1
data.val_mask = None
data.test_mask = torch.zeros(data.num_nodes, dtype=torch.bool)
data.test_mask[data.num_nodes - 500:] = 1
```
As I'm using the original split values.
But get 0.78 on the test set.
Should've I done something else? the link seems to follow exactly the implementation mentioned in the paper.

Thanks!

**Comment by dvirginz** [comment]
I'm sorry as it seems you follow a different split regime, which is different than the official cora regime. But if so, also unet archives 88 in my experiments under this regime, and in the comments here you  stated you were only able to achieve 89%

**Comment by rusty1s** [comment]
You are correct that we use a different split for Cora here, since `SplineConv` overfits when using the fixed split, but it starts outperforming GCN/Cheby when increasing the label rate.

**Comment by dvirginz** [comment]
Ok, and when you evaluated unet, was it with this split of the official one?
thanks:)

**Comment by rusty1s** [comment]
With the official one, as it is done in the paper :)

**Comment by dvirginz** [comment]
understood! thanks.

**Comment by rusty1s** [comment]
Oh really? I will check this.
